### DISCUSSION TITLE
Globalização/Tradução dos Componentes MudBlazor

### DISCUSSION THREAD

**Discussion by capneiva** [balta-io/forum#582] [Tira Dúvidas] [answered]
Curso = Fullstack .NET
Aula = MudDataGrid ToolBar

É possível traduzir os componentes do MudBlazor, como os Filros e etc. para o Português Brasil? Pois penso em utilizar em um pequeno projeto para um cliente, mas já tive problemas com outros clientes sobre o Idioma, principalmente para área de governos eetc., que é um item obrigatório no contrato e etc.

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @capneiva, como vai?

É possível sim, através deste item:
https://mudblazor.com/features/localization#resx-example

**Reply by capneiva** [reply]
Legal saber disso. Obrigado pela rápida resposta.

**Comment by capneiva** [comment]
Boa dia!

Tudo bem por aqui, espero que vocês estejam bem!

Legal que é possível... Obrigado pela rápida na resposta.



________________________________
